crates/core/tedge_api/src/topic.rs

```diff
@@ -78,6 +78,7 @@ pub fn get_child_id_from_child_topic(topic: &str) -> Option<String> {
 
 #[cfg(test)]
 mod tests {
+
```

Comment:
Remove this so that we can exclude this file from the change list.

Reply:
fixed